proof/crefine/lib/Corres_C.thy

```diff
@@ -101,6 +101,24 @@ lemma exvalue_liftxf [simp]:
   unfolding liftxf_def by simp
 
 
+\<comment> \<open>FIXME: move?\<close>
```

Comment:
Maybe to `lib/clib/CCorresLemmas.thy`? That's where the required `inl_rrel` and `inr_rrel` lemmas are.

Reply:
good plan

Reply:
Moved